tests/app/service/test_rest.py

```diff
@@ -3445,14 +3445,21 @@ def test_cancel_notification_for_service_raises_invalid_request_when_letter_is_i
     notification_status,
 ):
     sample_letter_notification.status = notification_status
+    sample_letter_notification.created_at = datetime.now()
 
     response = admin_request.post(
         'service.cancel_notification_for_service',
         service_id=sample_letter_notification.service_id,
         notification_id=sample_letter_notification.id,
         _expected_status=400
     )
-    assert response['message'] == 'It’s too late to cancel this letter. Printing started today at 5.30pm'
+    if notification_status == 'cancelled':
+        assert response['message'] == 'This latter has already been cancelled.'
```

Comment:
Optional: we could simplify this test to match the simplicity of the code e.g.

```
@pytest.mark.parametrize('status,expected_error', [
  ('cancelled', 'This letter...'),
  ('sent', 'We could not...')
])
```

I'm not sure what we gain from exhaustively testing every single possible status (apart from a lot more typing). But I appreciate this was there before and the PR is meant to be a quick fix, so happy to leave it as-is.

Reply:
cool, I will leave it as is :3. I guess what we would gain is failing tests if we changed which statuses are cancellable.